airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -86,7 +86,7 @@ class DAGDetailSchema(DAGSchema):
     doc_md = fields.String()
     default_view = fields.String()
     params = fields.Method('get_params', dump_only=True)
-    tags = fields.Method("get_tags", dump_only=True)
+    tags = fields.Method("get_tags", dump_only=True)  # type: ignore
```

Comment:
@mik-laj @ephraimbuddy - > I am not sure if this is a "real" bug or this assignment is valid here: 

The error here is:

```
airflow/api_connexion/schemas/dag_schema.py:89: error: Incompatible types in
assignment (expression has type "Method", base class "DAGSchema" defined the
type as "marshmallow.fields.List")
        tags = fields.Method("get_tags", dump_only=True)
```

Reply:
I think it's not a bug. We did it because the DAGDetailsSchema is not a database schema but object schema and there was no way we could extract the tags in that case

Reply:
Good one for `type: ignore` then

Reply:
cc @uranusjr